transformers/component_tests/triggers/vat_init_trigger_test.go

```diff
@@ -124,9 +123,13 @@ var _ = Describe("Updating historical_ilk_state table", func() {
 	})
 })
 
-func createVatInitModel(headerId, logId int64, ilkHex string) shared.InsertionModel {
+func createVatInitModel(headerId, logId int64, ilkHex string, database *postgres.DB) event.InsertionModel {
+
+	ilkID, ilkErr := shared.GetOrCreateIlk(ilkHex, database)
+	Expect(ilkErr).NotTo(HaveOccurred())
+
 	vatInit := test_data.VatInitModel
-	vatInit.ForeignKeyValues[constants.IlkFK] = ilkHex
+	vatInit.ColumnValues[constants.IlkColumn] = ilkID
 	vatInit.ColumnValues[constants.HeaderFK] = headerId
```

Comment:
Use `event` package constants for non-MCD variables (`headerFK`, `logFK`, `addressFK`) :) 

We'll remove these from the vdb-mcd-transformers repo later